internal/manager/config/config.go

```diff
@@ -50,7 +50,9 @@ const (
 
 	DefaultMaxSessionAge = 60 * 60 * 1 // 1 hours
 
-	Database = "database"
+	Database                 = "database"
+	DatabaseType             = "database_type"
```

Comment:
Couldn't this be inferred from the database connection string?

If it starts with `postgres://` then we use postgres. Otherwise, if it starts with `sqlite://` or is just a local file, use SQLite